templates/get_stock_movements/config.js

```diff
@@ -0,0 +1,15 @@
+config = {
```

Comment:
Please make sure this has a `name`, `description`, and `author`, otherwise it will not work. Please also make sure to use an appropriate title for the config section and not just "Custom Block"